_posts/2017-10-25-rails-girls-berlin-is-growing-up.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Rails Girls Berlin is growing up"
+date: "2017-10-25"
+coverImage: "IMG_0876b.jpg"
+---
+
+Rails Girls Berlin has existed since its first workshop in 2012. So far, we’ve organized more than 30 workshops with more than 1200 learners. From within our community, great projects such as [speakerinnen.org](http://speakerinnen.org), [rorganize.it](http://rorganize.it) and [diversitytickets.org](http://diversitytickets.org) have developed. Additionally, projects like [Rails Girls Summer of Code](https://railsgirlssummerofcode.org/), [ClojureBridge Berlin](http://clojurebridge-berlin.org) and [Heart of Code](http://heartofcode.org) are partially built by people who found or refreshed their love for programming with us.
+
+ 
+
+Five years of Rails Girls Berlin also means that our organizers team has had quite a few generations - which is to be expected when it comes to unpaid volunteer work. However, there is one discussion that popped up for each of these generations: it’s about our name. As organizers we have always tried to not speak of “girls” but of learners, coaches, organizers. Nonetheless, it happens that people from our community are referred to as girls, for which you can hardly blame anyone if you just look at our name.
+
+ 
+
+In general, we’re not angry about it. Girls are great and we’re happy to rock to Beyoncé’s “Who run the world? - Girls!”. The playful approach, the let’s-just-try-and-see perspective to coding is probably also associated to not being an adult. Beyond that, we’ve had the feeling that this name has helped to make us appear less threatening - maybe a reason why we have received a rather small amount of attacks or backlashes compared to other initiatives for marginalized groups in tech. Thus, we were able to inspire people to code and question stereotypes about tech without much interference.
+
+ 
+
+Nonetheless, our initiative is aimed towards adults. Mainly it’s for women and non-binary people and eventually, in theory, we would even accept men. However, with roughly 200 applications for 45 spots, they are given the lowest priority. But that’s not reflected in our name. Instead, we have school approaching us, asking if we could do a workshop for kids or teenagers. Don’t get us wrong - early tech education is important, but with the little resources we have, we need to stick to one target group.
```

Comment:
This is an example where women and non binary people are not the sole audience, https://berlin.rustbridge.com/events/nov-2019, so I think we can simplify this to

```
Nonetheless, our initiative is aimed towards adults. Historically for women and non-binary people. In the past we had schools approaching us, asking if we could do a workshop for kids or teenagers. We fully support younger education, but lack the resources or expertise to do so. We decided to be clearer in our naming, and be more explicit with each event who we are open to.